src/main/resources/config/application.properties

```diff
@@ -19,7 +19,7 @@ appeal.email.port=${SPRING_MAIL_PORT:25}
 appeal.email.smtp.tls.enabled=${SPRING_MAIL_PROPERTIES_MAIL_SMTP_STARTTLS_ENABLE:true}
 appeal.email.smtp.ssl.trust=${SPRING_MAIL_SMTP_SSL_TRUST:*}
 
-idam.s2s-auth.totp_secret=${IDAM_S2S_AUTH_TOTP_SECRET:J7PBT37Y6C3SEFGX}
+idam.s2s-auth.totp_secret=${IDAM_S2S_AUTH_TOTP_SECRET:secret}
```

Comment:
Is it safe to have this as part of the commit history? Might be worth rebasing if not

Reply:
This branch will be deleted.